SDK/Implement/SidechainSubWallet.cpp

```diff
@@ -53,11 +55,18 @@ namespace Elastos {
 
 			tx->SetTransactionType(Transaction::TransferCrossChainAsset, payload);
 
-			return tx->ToJson();
+			nlohmann::json txJson = tx->ToJson();
+			Log::retinfo("{} | {}", __FUNCTION__, txJson.dump());
+			return txJson;
 		}
 
 		std::string SidechainSubWallet::GetGenesisAddress() const {
-			return _config->GenesisAddress();
+			Log::preinfo("{}", __FUNCTION__);
```

Comment:
Any instance of subwallet should print the master and subwallet ID
Log::preinfo("{}:{} {}", _parent->GetId(), _info->GetChainID(), __FUNCTION__);